src/convertToObject.js

```diff
@@ -1,18 +1,32 @@
 'use strict';
 
+// Implement convertToObject function:
+// eslint-disable-next-line max-len
+// Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+// and returns an object where CSS properties are keys
+// eslint-disable-next-line max-len
+// and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCssObject = {};
+  const fromStringToArray = sourceString
+    .split(';')
+    .map(elem => elem.split(':'))
+    .filter(elem => elem.length !== 1)
+    .map(elem => elem.map(elem2 => elem2.trim()));
```

Comment:
It's been often said that we should not name variables like 'elem', maybe u need to replace it with something with more context

Reply:
Yeap, `elem` is a bad name for the variable